teleop/htm/templates/mobile_controller_ui.html

```diff
@@ -1,17 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" type="text/css" href="../static/CSS/mobileController.css">
   <link rel="shortcut icon" href="#" />
   <title>Mobile controller</title>
+  <style>
+    /* CSS styles for layout */
+    .mobile_view_container {
```

Comment:
Move your styles to the external .css file in static folder under `/htm/static/CSS/style.css`